core/templates/components/entity-creation-services/skill-creation.service.ts

```diff
@@ -41,18 +43,20 @@ angular.module('oppia').factory('SkillCreationService', [
         }
         skillCreationInProgress = true;
         AlertsService.clearWarnings();
-        $rootScope.loadingMessage = 'Creating skill';
         SkillCreationBackendApiService.createSkill(
           description, rubrics, explanation, linkedTopicIds)
           .then(function(response) {
             $timeout(function() {
-              $window.location = UrlInterpolationService.interpolateUrl(
-                CREATE_NEW_SKILL_URL_TEMPLATE, {
-                  skill_id: response.skillId
-                });
+              $rootScope.$broadcast(
```

Comment:
OK-ing this since the event already exists, but a note for the future that we are getting rid of $broadcast, so we'll need to find another way to do this. Can be done in separate PR though. (Ditto elsewhere.)

/cc @bansalnitish